frontend/sass/_accordion.scss

```diff
@@ -9,8 +9,8 @@
     p,
     summary {
       // Have border extend beyond element boundary for a better looking focus border
-      padding: 1rem;
-      margin: 0 -1rem;
+      padding: 0.5em 0.75rem;
```

Comment:
These seem like fairly minor changes, but since we do use this accordion in other places on the site, have we checked that it looks OK on, say, the NoRent FAQs page?

Reply:
yes! i clicked through a couple NoRent pages and it seemed reasonable to me, here are the FAQs:
<img width="288" alt="Screen Shot 2022-05-19 at 11 37 19 AM" src="https://user-images.githubusercontent.com/34112083/169375604-e2ee2280-19ef-4411-89f2-6a16a67d90e8.png">